apps/examples/react-ssr-example/src/client/index.tsx

```diff
@@ -21,14 +21,15 @@ const routes = Router.getConfig(routeConfig, config.publicPath);
 /**
  * Wrap application with container, router and store
  */
-const AppWrapper = () => (
-    <Provider store={configureStore("client")}>
-        <BrowserRouter>
-            <App routes={routes} />
-        </BrowserRouter>
-    </Provider>
-);
-
+function AppWrapper() {
```

Comment:
What's the difference here from the arrow function?

Reply:
in this particular case not much. This might be down to personal preference right now but arrow functions are not completely supported in all browsers so they need to be transpiled versus this does not (well except the JSX of course).

I think generally I am starting to only use arrow functions when "this" context is needed since it gets bound automatically. Or when we benefit from the short hand it can provide.